modules/flowable-engine/src/test/java/org/flowable/standalone/jpa/HistoricJPAVariableTest.java

```diff
@@ -91,8 +93,8 @@ public void testGetJPAEntityAsHistoricVariable() {
                 .processInstanceId(processInstanceId).variableName("simpleEntityFieldAccess").singleResult();
 
         Object value = historicVariableInstance.getValue();
-        assertTrue(value instanceof FieldAccessJPAEntity);
-        assertEquals(((FieldAccessJPAEntity) value).getValue(), simpleEntityFieldAccess.getValue());
+        assertThat(value instanceof FieldAccessJPAEntity).isTrue();
```

Comment:
use `assertThat(value).isInstanceOf(FieldAccessJPAEntity.class);`

Reply:
done; one that I missed :(